File: packages/server/src/__tests__/fixtures/python/Animal.py
"""
Animal base class for testing Python parser
"""


class Animal:
    """Base class for all animals"""
    
    def __init__(self, name: str, age: int):
        """Initialize animal with name and age"""
        self.name = name
        self.age = age
    
    def speak(self) -> str:
        """Make a sound"""
        return "Some sound"
    
    def get_name(self) -> str:
        """Get animal name"""
        return self.name

File: packages/server/src/__tests__/fixtures/java/Animal.java
package com.example;

/**
 * Animal base class for testing Java parser
 */
public class Animal {
    private String name;
    private int age;
    
    public Animal(String name, int age) {
        this.name = name;
        this.age = age;
    }
    
    public String speak() {
        return "Some sound";
    }
    
    public String getName() {
        return name;
    }
    
    public int getAge() {
        return age;
    }
}
